lnrpc/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.19.2-buster
+FROM golang:1.20.4-bullseye
```

Comment:
We need to stay at buster, otherwise the pinned versions of `protoc` won't be found, see CI failure.

Reply:
Sure, will change it to buster.